src/ontodia/viewUtils/connectionsMenu.tsx

```diff
@@ -25,11 +25,27 @@ const ALL_RELATED_ELEMENTS_LINK: FatLinkType = new FatLinkType({
     diagram: null,
 });
 
+export interface PropertySuggestionParams {
+    id: string;
+    key: string;
```

Comment:
The name `key` usually means something similar to `ID`. For search or filter text input the preferable name would be something like `token` or `text`.